src/cascadia/ut_app/JsonUtilsTests.cpp

```diff
@@ -318,9 +331,9 @@ namespace TerminalAppUnitTests
         // double -> float
         TryBasicType(1.0f, 1.0);
 
-        TryBasicType(til::color{ 0xab, 0xcd, 0xef }, "#abcdef");
+        TryBasicType(til::color{ 0xab, 0xcd, 0xef }, "#ABCDEF");
 
-        static const std::string testGuidString{ "{AA8147AA-E289-4508-BE83-FB68361EF2F3}" }; // can't use a string_view; jsoncpp hates it
+        static const std::string testGuidString{ "{aa8147aa-e289-4508-be83-fb68361ef2f3}" }; // can't use a string_view; jsoncpp hates it

```

Comment:
why?

Reply:
`::Microsoft::Console::Utils::GuidToString` outputs it as lowercase instead of uppercase.

Reply:
fine